### PR TITLE
Change to use kernel-slim image

### PR DESCRIPTION
**Pull request by gkwan-ibm**
Address issue OpenLiberty/guides-common/issues/861

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -111,6 +111,13 @@ You have access to the following projects and can switch between them with 'oc p
   * guide
 ```
 
+If your cluster does not install the https://cert-manager.io[cert-manager^] yet, install it by running the following command that refers to the official https://cert-manager.io/docs/installation[cert-manager installation] documentation:
```
Comment (jakub-pomykala) [inline README.adoc]: This doesn't make sense recommend:

```suggestion
If your cluster has not installed the https://cert-manager.io[cert-manager^] yet, install it by running the following command that refers to the official https://cert-manager.io/docs/installation[cert-manager installation] documentation:
```

**Review by jakub-pomykala — APPROVED** [review]
LGTM

File: README.adoc [hunk 2]
```diff
@@ -111,6 +111,13 @@ You have access to the following projects and can switch between them with 'oc p
   * guide
 ```
 
+If your cluster has not installed the https://cert-manager.io[cert-manager^] yet, install it by running the following command that refers to the official https://cert-manager.io/docs/installation[cert-manager installation] documentation:
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
If the https://cert-manager.io[cert-manager^]  is not installed on your cluster yet, install it by running the following command:
```

File: README.adoc [hunk 3]
```diff
@@ -111,6 +111,13 @@ You have access to the following projects and can switch between them with 'oc p
...
+[role='command']
+```
+oc apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
+```
+
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
For more information, see the https://cert-manager.io/docs/installation[cert-manager installation] documentation.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Pull request merged**